scripts/patch-utils.sh

```diff
@@ -97,22 +97,20 @@ function choose_kernel_branch {
 			exit 1
 			;;
 		esac
-	else
-		if [ "$2" != "focal" ]; 				# Ubuntu 20
-		then
-			echo -e "\e[31mUnsupported distribution $2, kernel version $1 . The patches are maintained for Ubuntu 16/18/20 LTS\e[0m" >&2
-			exit 1
-		fi
+	elif [ "$2" == "focal" ];
 		case "${kernel_version[0]}.${kernel_version[1]}" in
 		"5.4")									# kernel 5.4
-			echo master
+			echo hwe-5.4
```

Comment:
We'd want to stick for the master branch for users that use 5.4.
HWE currently leaped forward to 5.8 (Ubuntu-hwe-5.8-5.8.0-33.36_20.04.1), so if you can update the patches to match the changes then we'll continue to review and merge the PR